test/black-box/address/AddressAddCommand.test.ts

```diff
@@ -86,7 +87,7 @@ describe('AddressAddCommand', () => {
         res.expectJson();
         res.expectStatusCode(404);
         expect(res.error.error.success).toBe(false);
-        expect(res.error.error.message).toBe(`Entity with identifier Country code <TEST> was not valid! does not exist`);
+        expect(res.error.error.message).toBe(`Invalid profileId.`);
```

Comment:
one reason to create the exceptions was to have similar error messages everywhere, now if we change the error message in the exception, this hardcoded test will break... so, do something like this:
expect(res.error.error.message).toBe(new InvalidParamException('profileId').toString());